### PR TITLE
Fix function loading

### PR DESCRIPTION
**Pull request by kkuzmin**
### Solution Description
Fix library export to allow collector function to load using webpack.

Also fixes TMP directory usage for storing AlertLogic authentication tokens.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,11 +7,27 @@
...
-    Master: require('./master'),
-    Collector: require('./collector'),
-    Scaler: require('./scaler')
-};
+/**
+ * Master exports
+ * 
+ */
+var master = require('./master');
+exports.AlAzureMaster = master.AlAzureMaster;
```
Comment (mtrybulec) [inline index.js]: Not just require('./master').AlAzureMaster (and then can use the previous syntax do export everything in a single object def)?

File: master.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ class AlAzureMaster {
         };
         this._apiEndpoint = alApiEndpoint ? alApiEndpoint : process.env.CUSTOMCONNSTR_APP_AL_API_ENDPOINT;
         this._alAzcollectEndpoint = alAzcollectEndpoint ? alAzcollectEndpoint : process.env.APP_AZCOLLECT_ENDPOINT;
-        this._aimsc = new alcollector.AimsC(this._apiEndpoint, creds, undefined, MASTER_RETRY_OPTS);
+        this._aimsc = new alcollector.AimsC(this._apiEndpoint, creds, process.env.TMP, MASTER_RETRY_OPTS);
```
Comment (mtrybulec) [inline master.js]: Should the AimsC constructor also use process.env.TMP instead of DEFAULT_CACHE_DIR ('/tmp')?

File: index.js (hunk 1 above)
Comment (kkuzmin) [reply index.js]: I did it this way in case there will be more function/classes/constants exported.

File: master.js (hunk 2 above)
Comment (kkuzmin) [reply master.js]: I don't want to rely on `process.env` in `al-collector-js`. TMP directory path may come from configuration parameters of consumers. Going forward `al-aws-collector-js` and `al-azure-collector-js` use `process.env.TMP`

**Pull request merged**